controllers/common_constants.go

```diff
@@ -0,0 +1,5 @@
+package controllers
```

Comment:
Typically this file would be `common_consts.go` I think, not major, happy to keep as is, but I think that's the convention I've seen in the past